assets/xml/objects/gameplay_field_keep.xml

```diff
@@ -19,7 +19,7 @@
         <Limb Name="gameplay_field_keep_Standardlimb_002F60" Type="Standard" EnumName="GAMEPLAY_FIELD_KEEP_LIMB_05" Offset="0x2F60" />
         <Limb Name="gameplay_field_keep_Standardlimb_002F6C" Type="Standard" EnumName="GAMEPLAY_FIELD_KEEP_LIMB_06" Offset="0x2F6C" />
         <Limb Name="gameplay_field_keep_Standardlimb_002F78" Type="Standard" EnumName="GAMEPLAY_FIELD_KEEP_LIMB_07" Offset="0x2F78" />
-        <Skeleton Name="gameplay_field_keep_Skel_002FA0" Type="Normal" LimbType="Standard" LimbNone="GAMEPLAY_FIELD_KEEP_LIMB_NONE" LimbMax="GAMEPLAY_FIELD_KEEP_LIMB_MAX" EnumName="gameplay_field_keep_Limbs" Offset="0x2FA0" />
+        <Skeleton Name="gameplay_field_keep_Skel_002FA0" Type="Normal" LimbType="Standard" LimbNone="GAMEPLAY_FIELD_KEEP_LIMB_NONE" LimbMax="GAMEPLAY_FIELD_KEEP_LIMB_MAX" EnumName="GameplayFieldKeepLimb" Offset="0x2FA0" />
```

Comment:
Could you name the stuff that's used in EnButte? `gButterflySkel` etc. I make the exception here because it's the only unnamed skeleton in a keep object.

Reply:
I can, though I really do not want to do it in this PR. Seems better to just make a PR documenting `EnButte` instead (I'll finish what is in OoT and port it over). 
Though regarding your comment about this being the only unnamed skeleton in a keep object, there are actually 4 in gameplay_keep without autogenned enum names so are not  showing up in this PR. Out of curiosity @AngheloAlf was this intentional or an oversight? If an oversight I can add them to this PR real quick.

Reply:
So I went through and figured out what all the unnamed skeletons. I still feel naming everything associated with them is out of scope for this PR, instead I'll update the `EnumName` prefix for the buttefly, and add enums with the proper prefix for the gameplay_keep ones (as well as for the door skel which is named but has no enum).